javascript-forms/form-values.js

```diff
@@ -0,0 +1,22 @@
+var form = document.querySelector('form');
+
+form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var formData = {};
+  formData.name = form.elements.name.value;
+  formData.email = form.elements.email.value;
+  formData.message = form.elements.message.value;
+  console.log('Form data: ', formData);
+  form.reset();
+});
+
+// var nameValue = form.elements.name.value;
```

Comment:
When you are done with a feature, make sure you remove any commented code from your documents. Since the feature is complete and the code is still commented, that means it ended up not being needed, and should be removed 🤓 👍

Reply:
Oh yes, I saved it before I pushed it but I might need to save it before the terminal codes